pyvista/examples/examples.py

```diff
@@ -672,6 +673,183 @@ def _hydrogen_orbital_load_func(n=1, l=0, m=0, zoom_fac=1.0):
 _dataset_hydrogen_orbital = _DatasetLoader(_hydrogen_orbital_load_func)
 
 
+def load_snowflake(level: int = 6, as_triangles: bool = False):
+    """Load a Koch snowflake fractal.
+
+    The snowflake is generated recursively. By default, only the
+    outline is returned as a single :attr:`~pyvista.CellType.POLYLINE` cell.
+
+    This example is adapted from the VTK example `KochSnowflake
+    <https://examples.vtk.org/site/Python/Visualization/KochSnowflake/>`_.
+
+    For more information about the fractal, see:
+
+    - http://en.wikipedia.org/wiki/Koch_snowflake
+    - http://mathworld.wolfram.com/KochSnowflake.html.
+
+    Parameters
+    ----------
+    level : int, default: 6
+        Level of recursion to use.
+
+    as_triangles : bool, default: False
+        Return the snowflake as :attr:`~pyvista.CellType.TRIANGLE`
+        cells with an integer cell array ``'Level'`` indicating
+        the iteration level each triangle was generated.
+
+    Returns
+    -------
+    pyvista.PolyData
+        Snownflake mesh.
+
+    Examples
+    --------
+    Plot the default snowflake.
+
+    >>> from pyvista import examples
+    >>> snowflake = examples.load_snowflake()
+    >>> snowflake.plot(cpos='xy', line_width=5)
+
+    Plot the snowflake as triangles.
+
+    >>> snowflake = examples.load_snowflake(as_triangles=True)
+    >>> snowflake.plot(cpos='xy', cmap='blues', lighting=False)
+
+    .. seealso::
+
+        :ref:`Snowflake Dataset <snowflake_dataset>`
+            See this dataset in the Dataset Gallery for more info.
+
+    """
+    return _dataset_snowflake.load(level=level, as_triangles=as_triangles)
+
+
+def _snowflake_load_func(level=6, as_triangles=False):
+    from math import cos
+    from math import pi
+    from math import sin
+    from math import sqrt
+
+    def make_polyline(points, level_):
+        """Koch Snowflake as a vtkPolyLine."""
+        # Use the points from the previous iteration to create the points of the next
```

Comment:
This looks to be a significant copy and paste from the vtk-examples code.  Do we need to have a license associated with it then?  If we want to avoid this, we could recast the example to use PyVista code